lib/src/firestore.dart

```diff
@@ -681,9 +745,9 @@ class DocumentQuery {
   /// The [values] must be in order of [orderBy] filters.
   ///
   /// Cannot be used in combination with [startAt].
-  DocumentQuery startAfter(List<dynamic> values) {
-    final jsValues = jsify(values);
-    return new DocumentQuery(nativeInstance.startAfter(jsValues), firestore);
+  DocumentQuery startAfter({DocumentSnapshot snapshot, List<dynamic> values}) {
+    return new DocumentQuery(
+        _wrapPaginatingFunctionCall("startAfter", snapshot, values), firestore);
```

Comment:
Need to update dartdoc to something like this:

```
/// Takes a [snapshot] or a list of [values], creates and returns a new [DocumentQuery]
/// that starts after the provided fields relative to the order of the query.
///
/// Either [snapshot] or [values] can be provided at the same time, not both.
/// The [values] must be in order of [orderBy] filters.

/// Cannot be used in combination with [startAt].
```

Similar changes needed for other paginating methods.

Reply:
nit: Also might be a good idea to add an assert like:

```dart
assert(snapshot == null || values == null, 'Cannot provide both snapshot and values to startAfter.');
```

Reply:
ok